src/Persistence.php

```diff
@@ -196,40 +196,40 @@ public function typecastSaveRow(Model $m, array $row): array
      * may be "aliased" from SQL persistences or mapped depending on persistence
      * driver.
      */
-    public function typecastLoadRow(Model $m, array $row): array
+    public function typecastLoadRow(Model $model, array $row): array
     {
         $result = [];
-        foreach ($row as $key => $value) {
+        foreach ($row as $fieldName => $value) {
```

Comment:
is this correct?

I think we get persistenceName and save fieldName

Reply:
It is correct currently but in my opinion should be as you said.
Persistence returns the loaded data already with keys using short_name (implementing aliasing for Sql)
It is mostly useful in case field is an Sql expression.
I am thinking how to best modify it to be consistent and intuitive ...

Reply:
see how `typecastSaveRow` method is implemented

Reply:
I see. It should be same with load.
But internal workings of Sql persistence need to be changed a bit.
I will come up with a solution.
Add commit here or keep this PR only for introducing the method?

Reply:
both ways are fine with me, if you think, it is breaking a lot external apps, then 2nd PR